evap/staff/management/commands/run_tasks.py

```diff
@@ -26,12 +29,23 @@ def update_courses(self):
                     course.evaluation_end()
                     if course.is_fully_reviewed():
                         course.review_finished()
+                        if not course.is_graded:
+                            course.publish()
+                            evaluation_results_courses.append(course)
+                        elif course.final_grade_documents.exists():
+                            course.publish()
+                            grade_document_courses.append(course)
+                            evaluation_results_courses.append(course)
+                    elif course.final_grade_documents.exists():
+                        grade_document_courses.append(course)
```

Comment:
i initially thought this should go into the transition itself, but django-fsm looks like you can't do more transitions inside a transition.

...maybe the post_transition signal could be used? could you check how that would look like?